server/controllers/validators/messageControllerValidator.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @Author:     Alex Qin
+ * @Created:    2021.03.23
+ *
+ * @Description: functions to validate input to controller functions to send a message
+ *
+ */
+
+const { body } = require('express-validator/check');
+const {
+    isValidMessage,
+    uidShouldExistAndBeForAMember
+} = require('./messageControllerValidatorUtils');
+
+const messageContentValidation = [
+    body('content', "A message must be provided")
+        .exists()
+        .custom(content => isValidMessage(content)),
+    body('receiverId',"ReceiverId must be a user")
+        .exists()
```

Comment:
You should always try and check the data type as well. Can you check to make sure this is a number. See `userControllerValidator.js` to find an example of how I did this for a numeric field or look up the documentation for express-validator

Reply:
Addressed.